### PR TITLE
Kyria default reformat

### PR DESCRIPTION
**Pull request by precondition**
## Description

Reformat the keymap layout macro into a pretty, readable form (`qmk cformat` killed it) and add a README/write-up of the default Kyria keymap design

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/splitkb/kyria/keymaps/default/readme.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
+<h1 align="center"> Kyria's Default Keymap</h1>
```
Comment (fauxpark) [inline keyboards/splitkb/kyria/keymaps/default/readme.md:1]: Don't use HTML here.

File: keyboards/splitkb/kyria/keymaps/default/keymap.c [hunk 2]
```diff
@@ -37,136 +46,171 @@ enum layers { _QWERTY = 0, _DVORAK, _COLEMAK_DH, _NAV, _SYM, _FUNCTION, _ADJUST
 // produces the key `tap` when tapped (i.e. pressed and released).
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/default/keymap.c:48]: Just so it doesn't happen again...
```suggestion
// clang-format off
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

File: keyboards/splitkb/kyria/keymaps/default/keymap.c [hunk 3]
```diff
@@ -15,7 +15,16 @@
...
 
-enum layers { _QWERTY = 0, _DVORAK, _COLEMAK_DH, _NAV, _SYM, _FUNCTION, _ADJUST };
+enum layers {
+    _QWERTY = 0,
+    _DVORAK,
+    _COLEMAK_DH,
+    _NAV,
+    _SYM,
+    _FUNCTION,
+    _ADJUST
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/default/keymap.c:25]: This should prevent it from reformatting the enum, as well.
```suggestion
    _ADJUST,
```

Don't ask me why.... clang-format is ... weird.

File: keyboards/splitkb/kyria/keymaps/default/readme.md (hunk 1 above)
Comment (precondition) [reply keyboards/splitkb/kyria/keymaps/default/readme.md:1]: Can I keep the HTML formatting for the figure and KLE link or should that formatting be stripped too?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/splitkb/kyria/keymaps/default/readme.md (hunk 1 above)
Comment (drashna) [reply keyboards/splitkb/kyria/keymaps/default/readme.md]: Ideally, all HTML should be removed.

Comment (precondition) [reply keyboards/splitkb/kyria/keymaps/default/readme.md]: Solved in https://github.com/qmk/qmk_firmware/pull/14080/commits/dd1505e6c3dd74bfb6174ab602c9ace0b56f4750

**Review by drashna — APPROVED** [review]

File: keyboards/splitkb/kyria/keymaps/default/readme.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
...ef&fa@:0&:0&:0&:2%3B%3B&=N%0A%5D%0A%0ABrk%3B&@_y:-0.625&t=%23000000&fa@:5%3B%3B&=%E2%87%A7%0A%7C%0A%0A%0AShift&_f:3%3B&=Z%0A%5C%0AF1&_x:13&t=%23000000%0A%0A%0A%2300adef&f:3%3B&=%2F%2F%20%3F%0A%2F%2F%0A%0A%3Ci%20class%2F='kb%20kb-Multimedia-Mute-1'%3E%3C%2F%2Fi%3E&_t=%23000000&fa@:5&:0&:0&:2%3B%3B&=%E2%87%A7%0A%3F%0A%0APrtSc%0AShift%3B&@_y:-0.5&x:2.5&a:7&fa@:2%3B%3B&=ADJUST&_x:10&a:4&f:3%3B&=%E2%96%A4%3B&@_rx:4&ry:8.175&y:-4.675000000000001&x:-0.5&f:3%3B&=%3Ci%20class%2F='kb%20kb-logo-windows-8'%3E%3C%2F%2Fi%3E%0A%0A%0A%0AGUI%3B&@_rx:13&y:-4.675000000000001&x:-0.5&f:3%3B&=%3Ci%20class%2F='kb%20kb-logo-windows-8'%3E%3C%2F%2Fi%3E%0A%0A%0A%0AGUI%3B&@_r:15&rx:4&y:-4.675000000000001&x:-0.5&fa@:5%3B%3B&=%3Ci%20class%2F='kb%20kb-Return-2'%3E%3C%2F%2Fi%3E%0A%0A%0A%0AAlt%3B&@_r:30&y:-2&x:-0.5&f:3%3B&=%5B%20%7B%0A%7B%3B&@_x:-0.5&a:7%3B&=%3B&@_r:45&y:-2&x:-0.5&a:4&fa@:2&:0&:0&:2%3B%3B&=CapsLk%0A%0A%0AScrollLk%3B&@_x:-0.5&t=%23b09194&a:7&f:3%3B&=NAV%3B&@_r:-45&rx:13&y:-5.675000000000001&x:-0.5&t=%2315a398&f:3%3B&=F-keys%3B&@_x:-0.5&t=%230f7ba0&f:3%3B&=SYM%3B&@_r:-30&y:-2&x:-0.5&t=%23000000&a:4&f:3%3B&=%5D%20%7D%0A%7D%3B&@_x:-0.5&a:7%3B&=%3B&@_r:-15&y:-1&x:-0.5&a:4&f:3%3B&=AltGr%0A%0A%0A%0AAltGr
```
Comment (fauxpark) [inline keyboards/splitkb/kyria/keymaps/default/readme.md:7]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**